hphp/runtime/vm/jit/code-cache.cpp

```diff
@@ -97,7 +98,14 @@ CodeCache::CodeCache()
     exit(1);
```

Comment:
@mxw I tried to copy to functionality from here. Do you recommend printing the warning and not calling exit(1), or just removing this entire block?